src/systemd-rdma-settings.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+SERV=`echo $1 | awk -F@ '{print $1}'`
+FILE="/usr/lib/systemd/system/"$SERV"@.service"
+ARG_DIR=/run/systemd/system/$1
+NAME=$1
```

Comment:
The name `ARG_DIR` implies that the value is a directory, but it's a full path to a file, right?

How about this:

```
NAME=$1
CONF_FULL_PATH=/run/systemd/system/$NAME
```

and then s/ARG_DIR/CONF_FULL_PATH/g